### PR TITLE
 [init] Add missing help for args 

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

**Comment by MarcoFalke** [comment]
This also includes a bugfix:

screenshot from 2016-01-04 18-53-38
screenshot from 2016-01-04 18-53-22

**Comment by jonasschnelli** [comment]
utACK.
Nice catch (missing `uacomment` and maptxfee)!

**Comment by fanquake** [comment]
utACK

**Comment by laanwj** [comment]
Needs rebase after #7257

**Comment by MarcoFalke** [comment]
Rebased.

File: src/init.cpp [hunk 1]
```diff
@@ -453,6 +459,7 @@ std::string HelpMessage(HelpMessageMode mode)
     if (showDebug)
     {
         strUsage += HelpMessageOpt("-logtimemicros", strprintf("Add microsecond precision to debug timestamps (default: %u)", DEFAULT_LOGTIMEMICROS));
+        strUsage += HelpMessageOpt("-mocktime=<n>", "Replace actual time with <n> (default: 0)");
```
Comment (paveljanik) [inline src/init.cpp]: As `-mocktime=0` is a no op, I'd remove the default. What about this:
```
Replace actual time with <n> seconds since epoch
```

**Comment by paveljanik** [comment]
But ACK anyway.

File: src/init.cpp (hunk 1 above)
Comment (MarcoFalke) [inline src/init.cpp]: It's a developer documentation and I think it's useful to know that 0 means noop.

Though, I am using your extended description.

**Comment by MarcoFalke** [comment]
@laanwj Anything holding this back? (Some of this actually should go into 0.12.0, I can prepare the backport after you did the merge)

**Comment by jonasschnelli** [comment]
rc1 has be detached. I guess only important bugfixes should go into 0.12 now. But Re-ACK for master.

**Pull request merged**